ros2cli/ros2cli/node/direct.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+
+# TODO(mikaelarguedas) revisit this once it's specified
+HIDDEN_NODE_PREFIX = '_'
+
+DEFAULT_TIMEOUT = 0.1
```

Comment:
I didn't benchmark it but I often used more than a second on the non very beefy arm boards to be sure to have all topics. Then it will vary a lot according to the nature of the system so I'm fine with whatever value as long as it's easy to override it